init_pose: {
  x_pos: 0.4,
  y_pos: -0.22,
  z_pos: 0.63,
  x_orn: -0.5,
  y_orn: -0.5,
  z_orn: 0.5,
  w_orn: -0.5
}

table_pose: {
  x_pos: 0.4,
  y_pos: -0.5,
  z_pos: 0.3,
  x_orn: 0.0,
  y_orn: 0.0,
  z_orn: 0.7071068,
  w_orn: 0.7071068
}
table_dims: {
  x: 0.2,
  y: 0.4,
  z: 0.5
}
